packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -326,6 +329,18 @@ export interface ITelemetryContext {
      */
     set(prefix: string, property: string, value: TelemetryEventPropertyType): void;
 
+    /**
+     * Sets a set of values for telemetry data being tracked.
+     * @param prefix - unique prefix to tag this data with (ex: "fluid:summarize:")
+     * @param property - property name of the telemetry data being tracked (ex: "Options")
+     * @param values - A set of values to attribute to this summary telemetry data.
+     */
+    setAll(
```

Comment:
FYI - name is a bit weird. I'd rather prefer setMultiple().

Reply:
Maybe I can fix that in main and leave this be?

Reply:
https://github.com/microsoft/FluidFramework/pull/14154